src/main/java/teammates/common/datatransfer/FeedbackSessionType.java

```diff
@@ -1,8 +1,21 @@
 package teammates.common.datatransfer;
 
+/**
+ * An enum that defines the different types of feedback sessions.
+ */
 public enum FeedbackSessionType {
-        STANDARD, PRIVATE;
-        // TODO: be a bit more specific here.
-        // e.g., STANDARD_TEAM_PEER_EVAL, FULLY_CUSTOMIZABLE, PRIVATE, INSTRUCTOR_OBSERVATIONS
-        // Otherwise these will have to be renamed when we add more types later.
+    PRIVATE("PRIVATE"),
+    STANDARD("STANDARD"),
+    TEAM_EVALUATION("TEAMEVALUATION"),
+    OPTIMIZED_TEAM_EVALUATION("OPTIMIZEDTEAMEVALUATION");
```

Comment:
I think you can afford to add a static class in Const.java and put these session types in, so that in the future when you want to refer to these strings you can extract from the Const instead of using the getter method. (I feel it's a bit verbose)

Reply:
I thought about it as well. However, doing that would make this enum redundant because this enum is already meant to be a common resource (i.e. directly accessible from other classes), so it serves pretty much the same function as `Const.java`. 

What if I just shorten the getter method name instead (and the variable name) in `FeedbackSessionType.java`? Do you think that will work?

Reply:
Your currently approach tightly couples the `feedbackSessionType` with its corresponding string in the requests. However, you only use those strings when you want to distinguish which template the requests are asking for. Your current approach is fine, but to me there is too much unnecessary logics involved. To me, putting the session type strings inside Const and retrieving the enum type based on the comparison result looks more direct to me. Moreover, the feedback session type strings can be reused where necessary e.g. the status message. If so, then you do not need to put the fixed status message in Const.java anymore, but a template for those status messages (they look the same except the session names)